src/cli/onefuzz/templates/__init__.py

```diff
@@ -233,30 +233,38 @@ def wait_on(
         }
         self.wait_for_running = wait_for_running
 
-    def check_current_job(self) -> Job:
-        job = self.onefuzz.jobs.get(self.job.job_id)
-        if job.state in ["stopped", "stopping"]:
+    def check_current_job(self) -> List[Task]:
```

Comment:
i think this should be renamed to reflect the fact that we are getting the tasks
I propose 
`get_running_tasks_checked` or `check_running_job_tasks`